### ISSUE TITLE
FAQ

### ISSUE THREAD

**Issue by plfiorini** [lirios/lirios.github.io#33]
Explain common misconceptions to interested people.

**Comment by terencode** [comment]
Good idea,  maybe first explain Liri is a project unifying Hawaii and Papyros projects, thus already having some experience.

**Comment by TheCom3di4n** [comment]
(IMHO) It should be a part of the suppport page.
